src/components/dialogmote/referat/Deltakere.tsx

```diff
@@ -1,43 +1,67 @@
-import React from "react";
-import styled from "styled-components";
-import { Checkbox } from "nav-frontend-skjema";
+import React, { ReactElement } from "react";
 import { useVeilederinfo } from "../../../hooks/useVeilederinfo";
 import { useNavBrukerData } from "../../../data/navbruker/navbruker_hooks";
+import { Element } from "nav-frontend-typografi";
+import { Field } from "react-final-form";
+import { ReferatCheckbox } from "./ReferatCheckbox";
+import styled from "styled-components";
 
 const texts = {
-  deltakereTitle: "Deltakere i møtet",
+  title: "Deltakere i møtet",
 };
 
+export type Deltaker = "arbeidstaker" | "arbeidsgiver" | "veileder";
+
 interface DeltakereProps {
   arbeidsgiverNavn?: string;
 }
 
-const DeltakereBoks = styled.div`
-  margin-bottom: 2em;
+interface DeltakerFieldProps {
+  deltaker: Deltaker;
+  label: string;
+  disabled?: boolean;
+}
 
-  > * {
-    margin-bottom: 0.5em;
-  }
+const DeltakerField = ({
+  deltaker,
+  label,
+  disabled = false,
+}: DeltakerFieldProps) => (
+  <Field<Deltaker> name="deltakere" type="checkbox" value={deltaker}>
+    {({ input }) => (
+      <ReferatCheckbox {...input} label={label} disabled={disabled} />
+    )}
+  </Field>
+);
+
+const DeltakereBoks = styled.div`
+  margin-bottom: 4em;
 `;
 
-const BoldText = styled.p`
-  font-weight: bold;
+const Header = styled(Element)`
+  margin-bottom: 1em;
 `;
 
-const Deltakere = ({ arbeidsgiverNavn }: DeltakereProps) => {
+const Deltakere = ({ arbeidsgiverNavn }: DeltakereProps): ReactElement => {
   const navbruker = useNavBrukerData();
   const { veilederinfo } = useVeilederinfo();
 
   return (
     <DeltakereBoks>
-      <BoldText>{texts.deltakereTitle}</BoldText>
-      <Checkbox label={`Arbeidstager: ${navbruker?.navn}`} checked readOnly />
-      <Checkbox
+      <Header>{texts.title}</Header>
+      <DeltakerField
```

Comment:
Disabled checkbox som liste-element bør unngås, det er vel ikke innafor mtp universell utforming.

Man brukes også samme checkbox-visning lenger nede på siden, og der er den klikkbar. Kan nok være forvirrende.

Kan gjerne høre med Espen om hva han tenker?

Reply:
Ja, i følge Espen er det best om arbeidstaker ikke kan krysses bort som deltaker (da skal møtet avlyses). Kan ta en runde på om vi kan/bør vise det fram på en annen måte. Kunne f.eks. lagt på en https://design.nav.no/components/hjelpetekst som sier hvorfor den er inaktiv.

Reply:
Det mest "riktige" er vel å tillate klikking på alt, men gi en god feilmelding når veilederen gjør noe vi ikke vil 👍

Reply:
👍 Snakket med Espen også, så vi går for å ha aktiv checkbox (legger på validering senere).